src/main/java/com/suse/salt/netapi/exception/SaltUserUnauthorizedException.java

```diff
@@ -6,6 +6,8 @@
  */
 public class SaltUserUnauthorizedException extends SaltException {
 
+    private static final long serialVersionUID = 4L;
```

Comment:
I read that earlier, and thought it'd be a good idea to add the serialVersionUID not to avoid the warnings in IDEs, but to avoid compile-time errors as outlined in the post.  Perhaps a utility function could be written instead to generate unique longs instead of leaving it up to the Java compiler.